draft-ietf-ice-rfc5245bis.xml

```diff
@@ -876,10 +850,15 @@ following exceptions:
     interface, and are part of the same network prefix MUST NOT be
     gathered.</t>
 
-    <t>Host candidates corresponding to IPv6 link-local addresses MUST NOT 
-    be gathered.</t>
+    <t>If one or more host candidates corresponding to an IPv6 
+    address generated using a mechanism that prevents location 
+    tracking <xref target="RFC7721"/> are gathered, host candidates 
+    corresponding to IPv6 addresses that do allow location tracking 
+    that are configured on the same interface and are part of the 
+    same network prefix MUST NOT be gathered; and host candidates 
+    corresponding to IPv6 link-local addresses <xref target="RFC4291"/>
+    MUST NOT be gathered.</t>
```

Comment:
Does this need to get run by the WG?

Reply:
I'll let the chairs deal with that, but this text was already in an earlier version of the draft.